src/PriceOracle.sol

```diff
@@ -101,7 +108,10 @@ contract PriceOracle is AccessControl {
 
             uint256 multiplier = minimumPrice;
             uint256 usedRedundancy = redundancy;
+            uint256 currentRoundNumber = currentRound();
 
+            // price can only be adjusted once per round
+            require(currentRoundNumber != lastClaimedRound, "price already adjusted in this round");
```

Comment:
Should this be != or >?  There is logic below that will get a negative rounds skipped if the round is ever allowed to go backwards.

Reply:
Primary purpose of this was to not let the possibility to call adjustPrice in the same round more than once, hence the message.

">" would do the same but if we manage to get "currentRoundNumber" to be lower than "lastClaimedRound"
we do have a set of some other problems :)